test/test_nn.py

```diff
@@ -1800,6 +1800,10 @@ def add_test(test):
         test_cuda=False
     ),
     dict(
+        constructor=lambda: nn.CosineSimilarity(),
```

Comment:
Right, uhh might I ask how to pass to the testing framework two parameters for the `forward` method ?
`input=(torch.rand(5,7),torch.rand(5,7))` does not work, nor does it specifying `input_size=(2,5,7)`.